diff -up chromium-81.0.4044.9/media/gpu/vaapi/BUILD.gn.24~ chromium-81.0.4044.9/media/gpu/vaapi/BUILD.gn
--- chromium-81.0.4044.9/media/gpu/vaapi/BUILD.gn.24~	2020-02-05 22:54:12.000000000 +0100
+++ chromium-81.0.4044.9/media/gpu/vaapi/BUILD.gn	2020-02-09 23:33:25.660636399 +0100
@@ -13,6 +13,10 @@ import("//ui/ozone/ozone.gni")
 assert(is_linux)
 assert(use_vaapi)
 
+config("vaapi_permissive") {
+  cflags = [ "-fpermissive" ]
+}
+
 generate_stubs("libva_stubs") {
   extra_header = "va_stub_header.fragment"
   sigs = [ "va.sigs" ]
@@ -119,6 +123,8 @@ source_set("vaapi") {
     ]
   }
 
+  configs += [ ":vaapi_permissive" ]
+
   if (use_x11) {
     configs += [ "//build/config/linux:x11" ]
     deps += [ "//ui/gfx/x" ]
