diff -up chromium-90.0.4412.3/chrome/browser/profiling_host/BUILD.gn.omv~ chromium-90.0.4412.3/chrome/browser/profiling_host/BUILD.gn
--- chromium-90.0.4412.3/chrome/browser/profiling_host/BUILD.gn.omv~	2021-02-12 16:57:09.254914601 +0100
+++ chromium-90.0.4412.3/chrome/browser/profiling_host/BUILD.gn	2021-02-12 16:57:18.958931656 +0100
@@ -24,7 +24,6 @@ static_library("profiling_host") {
     "//components/version_info",
     "//content/public/browser",
     "//content/public/common",
-    "//third_party/zlib:zlib_common_headers",
 
     # Added to support the dependency on //chrome/browser/metrics
     # TODO(darin): We should instead have a way to express that dependency here.
