diff -up chromium-94.0.4603.0/chrome/browser/profiling_host/BUILD.gn.28~ chromium-94.0.4603.0/chrome/browser/profiling_host/BUILD.gn
--- chromium-94.0.4603.0/chrome/browser/profiling_host/BUILD.gn.28~	2021-08-16 05:31:17.475647921 +0200
+++ chromium-94.0.4603.0/chrome/browser/profiling_host/BUILD.gn	2021-08-16 05:34:11.071294405 +0200
@@ -22,7 +22,6 @@ static_library("profiling_host") {
     "//components/version_info",
     "//content/public/browser",
     "//content/public/common",
-    "//third_party/zlib:zlib",
 
     # Added to support the dependency on //chrome/browser/metrics
     # TODO(darin): We should instead have a way to express that dependency here.
