--- chromium-53.0.2756.0/build/common.gypi.0022~	2016-06-03 12:55:28.382786198 +0200
+++ chromium-53.0.2756.0/build/common.gypi	2016-06-03 13:00:05.927663811 +0200
@@ -6069,13 +6069,6 @@
     }],
     ['use_lto==1 and clang==1', {
       'target_defaults': {
-        'target_conditions': [
-          ['_toolset=="target"', {
-            'arflags': [
-              '--plugin', '../../<(make_clang_dir)/lib/LLVMgold.so',
-            ],
-          }],
-        ],
         'msvs_settings': {
           'VCCLCompilerTool': {
             'AdditionalOptions': [
@@ -6264,7 +6257,7 @@
       },
     }],
     # TODO(pcc): Make these flags work correctly with CFI.
-    ['use_lto!=0 and cfi_vptr==0', {
+    ['use_lto!=0 and cfi_vptr==0 and clang==0', {
       'target_defaults': {
         'target_conditions': [
           ['_toolset=="target"', {
--- chromium-53.0.2756.0/build/config/compiler/BUILD.gn.0022~	2016-06-03 06:40:48.000000000 +0200
+++ chromium-53.0.2756.0/build/config/compiler/BUILD.gn	2016-06-03 12:55:28.386786197 +0200
@@ -465,7 +465,7 @@ config("compiler") {
     }
 
     # TODO(pcc): Make these flags work correctly with CFI.
-    if (!is_cfi) {
+    if (!is_cfi && !is_clang) {
       cflags += [ "-fwhole-program-vtables" ]
       ldflags += [ "-fwhole-program-vtables" ]
     }
