diff -up chromium-99.0.4818.0/tools/gn/build/gen.py.11~ chromium-99.0.4818.0/tools/gn/build/gen.py
--- chromium-99.0.4818.0/tools/gn/build/gen.py.11~	2022-01-12 14:27:02.931540932 +0100
+++ chromium-99.0.4818.0/tools/gn/build/gen.py	2022-01-12 14:29:46.159571709 +0100
@@ -171,7 +171,7 @@ def main(argv):
   args_list.add('--no-strip', action='store_true',
                     help='Don\'t strip release build. Useful for profiling.')
   args_list.add('--no-static-libstdc++', action='store_true',
-                    default=False, dest='no_static_libstdcpp',
+                    default=True, dest='no_static_libstdcpp',
                     help='Don\'t link libstdc++ statically')
   args_list.add('--link-lib',
                     action='append',
