--- chromium-64.0.3253.3/components/machine_intelligence/generic_logistic_regression_inference.cc.omv~	2017-11-06 03:42:02.316317130 +0100
+++ chromium-64.0.3253.3/components/machine_intelligence/generic_logistic_regression_inference.cc	2017-11-06 03:42:13.874347657 +0100
@@ -6,6 +6,7 @@
 
 #include "base/logging.h"
 #include "components/machine_intelligence/ranker_example_util.h"
+#include <cmath>
 
 namespace machine_intelligence {
 
