--- chromium-65.0.3311.3/third_party/angle/src/vulkan_support/BUILD.gn.omv~	2018-01-06 17:28:49.405477592 +0100
+++ chromium-65.0.3311.3/third_party/angle/src/vulkan_support/BUILD.gn	2018-01-06 17:33:34.230502788 +0100
@@ -149,8 +149,6 @@ foreach(script_and_dep, helper_script_an
 action("spirv_tools_external_revision_generate") {
   script = "$vulkan_layers_dir/scripts/external_revision_generator.py"
   inputs = [
-    "$spirv_tools_dir/.git/HEAD",
-    "$spirv_tools_dir/.git/index",
   ]
   outputs = [
     "$vulkan_gen_dir/spirv_tools_commit_id.h",
