diff -up chromium-88.0.4300.0/third_party/dawn/src/common/ityp_array.h.omv~ chromium-88.0.4300.0/third_party/dawn/src/common/ityp_array.h
--- chromium-88.0.4300.0/third_party/dawn/src/common/ityp_array.h.omv~	2020-11-05 02:32:57.027342875 +0100
+++ chromium-88.0.4300.0/third_party/dawn/src/common/ityp_array.h	2020-11-05 02:33:01.565362530 +0100
@@ -27,7 +27,7 @@ namespace ityp {
     // indices must be a particular type |Index|. Dawn uses multiple flat maps of
     // index-->data, and this class helps ensure an indices cannot be passed interchangably
     // to a flat map of a different type.
-    template <typename Index, typename Value, size_t Size>
+    template <typename Index, typename Value, std::size_t Size>
     class array : private std::array<Value, Size> {
         using I = UnderlyingType<Index>;
         using Base = std::array<Value, Size>;
