--- chromium-64.0.3253.3/third_party/pdfium/pdfium.gni.omv~	2017-11-07 15:13:23.306021552 +0100
+++ chromium-64.0.3253.3/third_party/pdfium/pdfium.gni	2017-11-07 15:12:42.546003860 +0100
@@ -61,6 +61,9 @@ declare_args() {
 
   # Don't build against bundled lcms2.
   use_system_lcms2 = false
+
+  # Don't build against bundled libpng.
+  use_system_libpng = false
 }
 
 if (pdf_use_skia && pdf_use_skia_paths) {
--- chromium-64.0.3253.3/third_party/pdfium/BUILD.gn.omv~	2017-11-07 15:13:23.306021552 +0100
+++ chromium-64.0.3253.3/third_party/pdfium/BUILD.gn	2017-11-07 15:12:42.546003860 +0100
@@ -779,7 +779,7 @@ static_library("fxcodec") {
       "core/fxcodec/lbmp/fx_bmp.h",
     ]
     deps += [
-      "third_party:fx_lpng",
+      "third_party:png",
       "third_party:fx_tiff",
     ]
   }
--- chromium-64.0.3253.3/third_party/pdfium/third_party/BUILD.gn.omv~	2017-11-07 15:05:54.043831465 +0100
+++ chromium-64.0.3253.3/third_party/pdfium/third_party/BUILD.gn	2017-11-07 15:09:13.025901598 +0100
@@ -398,6 +398,22 @@ static_library("fx_libopenjpeg") {
   ]
 }
 
+if (use_system_libpng) {
+  pkg_config("libpng_from_pkgconfig") {
+    defines = [ "USE_SYSTEM_LIBPNG" ]
+    packages = [ "libpng" ]
+  }
+}
+group("png") {
+  if (use_system_libpng) {
+    public_configs = [ ":libpng_from_pkgconfig" ]
+  } else {
+    public_deps = [
+      ":fx_lpng",
+    ]
+  }
+}
+
 static_library("fx_lpng") {
   configs -= [ "//build/config/compiler:chromium_code" ]
   configs += [
