--- chromium-51.0.2687.0/chrome/browser/ui/webui/engagement/site_engagement_ui.cc.omv~	2016-03-29 16:22:04.002376651 +0200
+++ chromium-51.0.2687.0/chrome/browser/ui/webui/engagement/site_engagement_ui.cc	2016-03-29 16:22:14.204256155 +0200
@@ -55,7 +55,7 @@ class SiteEngagementUIHandlerImpl : publ
                                        double score) override {
     GURL origin_gurl(origin.get());
     if (!origin_gurl.is_valid() || score < 0 ||
-        score > SiteEngagementScore::kMaxPoints || isnan(score)) {
+        score > SiteEngagementScore::kMaxPoints || std::isnan(score)) {
       return;
     }
 
