diff -up chromium-70.0.3521.2/tools/gn/build/gen.py.omv~ chromium-70.0.3521.2/tools/gn/build/gen.py
--- chromium-70.0.3521.2/tools/gn/build/gen.py.omv~	2018-08-16 21:24:08.588635066 +0200
+++ chromium-70.0.3521.2/tools/gn/build/gen.py	2018-08-16 21:24:18.432681991 +0200
@@ -368,7 +368,6 @@ def WriteGNNinja(path, platform, host, o
         cflags.append('--sysroot=' + linux_sysroot)
         ldflags.append('--sysroot=' + linux_sysroot)
       ldflags.extend([
-          '-static-libstdc++',
           '-Wl,--as-needed',
       ])
       libs.extend([
diff -up chromium-70.0.3521.2/tools/gn/infra/recipes/gn.py.omv~ chromium-70.0.3521.2/tools/gn/infra/recipes/gn.py
--- chromium-70.0.3521.2/tools/gn/infra/recipes/gn.py.omv~	2018-08-16 21:24:30.182737874 +0200
+++ chromium-70.0.3521.2/tools/gn/infra/recipes/gn.py	2018-08-16 21:24:38.336776577 +0200
@@ -61,7 +61,7 @@ def RunSteps(api, repository):
           'CC': cipd_dir.join('bin', 'clang'),
           'CXX': cipd_dir.join('bin', 'clang++'),
           'AR': cipd_dir.join('bin', 'llvm-ar'),
-          'LDFLAGS': '-static-libstdc++ -ldl -lpthread',
+          'LDFLAGS': '-ldl -lpthread',
       },
       'mac': {},
       'win': {},
