--- chromium-52.0.2743.6/third_party/pdfium/xfa/fxbarcode/utils.h.omv~	2016-05-25 20:37:17.560466786 +0200
+++ chromium-52.0.2743.6/third_party/pdfium/xfa/fxbarcode/utils.h	2016-05-25 20:37:38.984389995 +0200
@@ -26,7 +26,7 @@ void BC_FX_ByteString_Append(CFX_ByteStr
 #endif
 #if (_FX_OS_ == _FX_WIN32_DESKTOP_ || _FX_OS_ == _FX_WIN64_)
 #define FXSYS_isnan(x) _isnan(x)
-#elif(_FX_OS_ == _FX_MACOSX_ || _FX_OS_ == _FX_IOS_)
+#elif(_FX_OS_ == _FX_MACOSX_ || _FX_OS_ == _FX_IOS_ || defined(__clang__))
 #include <cmath>
 #define FXSYS_isnan(x) std::isnan(x)
 #elif(_FX_OS_ == _FX_LINUX_DESKTOP_ || _FX_OS_ == _FX_ANDROID_)
