diff -up chromium-69.0.3472.3/tools/gn/BUILD.gn.0021~ chromium-69.0.3472.3/tools/gn/BUILD.gn
--- chromium-69.0.3472.3/tools/gn/BUILD.gn.0021~	2018-07-06 22:10:17.323716477 +0200
+++ chromium-69.0.3472.3/tools/gn/BUILD.gn	2018-07-06 22:11:45.216719755 +0200
@@ -268,7 +268,6 @@ executable("gn") {
 
   deps = [
     ":gn_lib",
-    ":last_commit_position",
     "//base",
     "//build/win:default_exe_manifest",
   ]
diff -up chromium-69.0.3472.3/tools/gn/gn_main.cc.0021~ chromium-69.0.3472.3/tools/gn/gn_main.cc
--- chromium-69.0.3472.3/tools/gn/gn_main.cc.0021~	2018-06-26 06:33:22.000000000 +0200
+++ chromium-69.0.3472.3/tools/gn/gn_main.cc	2018-07-06 22:10:17.323716477 +0200
@@ -19,13 +19,7 @@
 #include "tools/gn/standard_out.h"
 #include "tools/gn/switches.h"
 
-// Only the GN-generated build makes this header for now.
-// TODO(brettw) consider adding this if we need it in GYP.
-#if defined(GN_BUILD)
-#include "tools/gn/last_commit_position.h"
-#else
 #define LAST_COMMIT_POSITION "UNKNOWN"
-#endif
 
 namespace {
 
