--- chromium-49.0.2612.0/ui/base/resource/data_pack.cc.omv~	2016-01-09 14:16:42.702251081 +0100
+++ chromium-49.0.2612.0/ui/base/resource/data_pack.cc	2016-01-09 14:16:47.371252376 +0100
@@ -157,7 +157,7 @@ bool DataPack::LoadImpl() {
   // 2) Verify the entries are within the appropriate bounds. There's an extra
   // entry after the last item which gives us the length of the last item.
   for (size_t i = 0; i < resource_count_ + 1; ++i) {
-    uint32 t;
+    uint32_t t;
     const DataPackEntry* entry = reinterpret_cast<const DataPackEntry*>(
         mmap_->data() + kHeaderLength + (i * sizeof(DataPackEntry)));
     memcpy(&t, &entry->file_offset, 32/8);
