diff -up chromium-81.0.4044.9/chrome/browser/about_flags.cc.23~ chromium-81.0.4044.9/chrome/browser/about_flags.cc
--- chromium-81.0.4044.9/chrome/browser/about_flags.cc.23~	2020-02-05 22:54:11.000000000 +0100
+++ chromium-81.0.4044.9/chrome/browser/about_flags.cc	2020-02-09 23:15:10.037708142 +0100
@@ -1892,7 +1892,7 @@ const FeatureEntry kFeatureEntries[] = {
         "disable-accelerated-video-decode",
         flag_descriptions::kAcceleratedVideoDecodeName,
         flag_descriptions::kAcceleratedVideoDecodeDescription,
-        kOsMac | kOsWin | kOsCrOS | kOsAndroid,
+        kOsMac | kOsWin | kOsCrOS | kOsAndroid | kOsLinux,
         SINGLE_DISABLE_VALUE_TYPE(switches::kDisableAcceleratedVideoDecode),
     },
     {
@@ -2324,12 +2324,12 @@ const FeatureEntry kFeatureEntries[] = {
      FEATURE_VALUE_TYPE(service_manager::features::kXRSandbox)},
 #endif  // !defined(OS_ANDROID)
 #endif  // ENABLE_VR
-#if defined(OS_CHROMEOS)
+#if defined(OS_CHROMEOS) || defined(OS_LINUX)
     {"disable-accelerated-mjpeg-decode",
      flag_descriptions::kAcceleratedMjpegDecodeName,
-     flag_descriptions::kAcceleratedMjpegDecodeDescription, kOsCrOS,
+     flag_descriptions::kAcceleratedMjpegDecodeDescription, kOsCrOS | kOsLinux,
      SINGLE_DISABLE_VALUE_TYPE(switches::kDisableAcceleratedMjpegDecode)},
-#endif  // OS_CHROMEOS
+#endif  // OS_CHROMEOS || OS_LINUX
     {"system-keyboard-lock", flag_descriptions::kSystemKeyboardLockName,
      flag_descriptions::kSystemKeyboardLockDescription, kOsDesktop,
      FEATURE_VALUE_TYPE(features::kSystemKeyboardLock)},
diff -up chromium-81.0.4044.9/chrome/browser/flag_descriptions.cc.23~ chromium-81.0.4044.9/chrome/browser/flag_descriptions.cc
--- chromium-81.0.4044.9/chrome/browser/flag_descriptions.cc.23~	2020-02-05 22:54:11.000000000 +0100
+++ chromium-81.0.4044.9/chrome/browser/flag_descriptions.cc	2020-02-09 23:15:10.038708142 +0100
@@ -2972,16 +2972,20 @@ const char kMetalDescription[] =
 
 #endif
 
-// Chrome OS -------------------------------------------------------------------
+// Chrome OS & Linux -----------------------------------------------------------
 
-#if defined(OS_CHROMEOS)
+#if defined(OS_CHROMEOS) || (defined(OS_LINUX) && !defined(OS_ANDROID))
 
 const char kAcceleratedMjpegDecodeName[] =
     "Hardware-accelerated mjpeg decode for captured frame";
 const char kAcceleratedMjpegDecodeDescription[] =
     "Enable hardware-accelerated mjpeg decode for captured frame where "
     "available.";
+#endif
 
+// Chrome OS -------------------------------------------------------------------
+
+#if defined(OS_CHROMEOS)
 const char kAggregatedMlAppRankingName[] = "Rank suggested apps with ML.";
 const char kAggregatedMlAppRankingDescription[] =
     "Use the aggregated ML model to rank the suggested apps.";
diff -up chromium-81.0.4044.9/chrome/browser/flag_descriptions.h.23~ chromium-81.0.4044.9/chrome/browser/flag_descriptions.h
--- chromium-81.0.4044.9/chrome/browser/flag_descriptions.h.23~	2020-02-05 22:54:11.000000000 +0100
+++ chromium-81.0.4044.9/chrome/browser/flag_descriptions.h	2020-02-09 23:15:10.039708142 +0100
@@ -1717,13 +1717,17 @@ extern const char kMetalDescription[];
 
 #endif  // defined(OS_MACOSX)
 
-// Chrome OS ------------------------------------------------------------------
+// Chrome OS & Linux ----------------------------------------------------------
 
-#if defined(OS_CHROMEOS)
+#if defined(OS_CHROMEOS) || (defined(OS_LINUX) && !defined(OS_ANDROID))
 
 extern const char kAcceleratedMjpegDecodeName[];
 extern const char kAcceleratedMjpegDecodeDescription[];
 
+#endif
+
+// Chrome OS ------------------------------------------------------------------
+#if defined(OS_CHROMEOS)
 extern const char kAggregatedMlAppRankingName[];
 extern const char kAggregatedMlAppRankingDescription[];
 
diff -up chromium-81.0.4044.9/gpu/config/software_rendering_list.json.23~ chromium-81.0.4044.9/gpu/config/software_rendering_list.json
--- chromium-81.0.4044.9/gpu/config/software_rendering_list.json.23~	2020-02-09 23:15:10.040708142 +0100
+++ chromium-81.0.4044.9/gpu/config/software_rendering_list.json	2020-02-09 23:25:04.617669208 +0100
@@ -337,7 +337,8 @@
     },
     {
       "id": 48,
-      "description": "Accelerated video decode is unavailable on Linux",
+      "description": "Accelerated VA-API video decode is not supported on NVIDIA platforms",
+      "vendor_id": "0x10de",
       "cr_bugs": [137247, 1032907],
       "os": {
         "type": "linux"
